db/db/src/main/java/io/helidon/db/DbExecute.java

```diff
@@ -0,0 +1,407 @@
+    default CompletionStage<Long> namedInsert(String statementName, Object... parameters) {
+        return createNamedInsert(statementName).params(parameters).execute();
+    }
+
+    /**
+     * Create and execute insert statement using a statement passed as an argument.
+     *
+     * @param statement  the insert statement to be executed
+     * @param parameters query parameters to set
+     * @return number of rows inserted into the database
+     */
+    default CompletionStage<Long> insert(String statement, Object... parameters) {
+        return createInsert(statement).params(parameters).execute();
+    }
+
+    /*
+     * UPDATE
+     */
+
+    /**
+     * Create an update statement using a named statement passed as an argument.
+     *
+     * @param statementName the name of the statement
+     * @param statement the statement text
+     * @return database statement that can update data
+     */
+    default DbStatement<?, CompletionStage<Long>> createNamedUpdate(String statementName, String statement) {
+        return createNamedDmlStatement(statementName, statement);
+    }
+
+    /**
+     * Create an update statement using a named statement.
+     *
+     * @param statementName the name of the statement
+     * @return database statement that can update data
+     */
+    DbStatement<?, CompletionStage<Long>> createNamedUpdate(String statementName);
+
+    /**
+     * Create an update statement using a statement text.
+     *
+     * @param statement the statement text
+     * @return database statement that can update data
+     */
+    DbStatement<?, CompletionStage<Long>> createUpdate(String statement);
+
+    /**
+     * Create and execute update statement using a statement defined in the configuration file.
+     *
+     * @param statementName the name of the configuration node with statement
+     * @param parameters    query parameters to set
+     * @return number of rows updateed into the database
+     */
+    default CompletionStage<Long> namedUpdate(String statementName, Object... parameters) {
+        return createNamedUpdate(statementName).params(parameters).execute();
+    }
+
+    /**
+     * Create and execute update statement using a statement passed as an argument.
+     *
+     * @param statement  the update statement to be executed
+     * @param parameters query parameters to set
+     * @return number of rows updateed into the database
+     */
+    default CompletionStage<Long> update(String statement, Object... parameters) {
+        return createUpdate(statement).params(parameters).execute();
+    }
+
+    /*
+     * DELETE
+     */
+
+    /**
+     * Create a delete statement using a named statement passed as an argument.
+     *
+     * @param statementName the name of the statement
+     * @param statement the statement text
+     * @return database statement that can delete data
+     */
+    default DbStatement<?, CompletionStage<Long>> createNamedDelete(String statementName, String statement) {
```

Comment:
Another occurence of _hybrid_ method with both statement name and statement string.